test/test_helper.rb

```diff
@@ -1,3 +1,12 @@
+require 'simplecov'
+SimpleCov.start do
+  add_group 'Controllers', 'app/controllers'
+  add_group 'Helpers', 'app/helpers'
+  add_group 'Mailers', 'app/mailers'
+  add_group 'Lib', 'lib'
+  add_group 'Models', 'app/models'
+end
```

Comment:
@vlasiak, OK. What about benefits of usage of [predefined](https://github.com/colszowka/simplecov#profiles) `rails` profile?  Doesn't it do the same configuration (and even more)? What do you think?